docs/game/ship.md

```diff
@@ -27,30 +27,51 @@ When using ShipThis in a CI environment, it is most useful to use the `--follow`
 
 [![asciicast](https://asciinema.org/a/gKmZ0E1rJ4oiT9SyuSivXBZfY.svg)](https://asciinema.org/a/gKmZ0E1rJ4oiT9SyuSivXBZfY#shipthis-col80row24)
 
-### Advanced usage - follow, do not publish, and then download APK
+### Follow, do not publish, and then download APK
 
 [![asciicast](https://asciinema.org/a/GNf0t8niOlrMDsgPKqmBcuqQh.svg)](https://asciinema.org/a/GNf0t8niOlrMDsgPKqmBcuqQh#shipthis-col80row24)
 
+### Building with demo credentials
+
+Adding the `--useDemoCredentials` flag executes the build for the specified platform but applies ShipThis certificates, keystores or provisioning profiles. This can be useful to generate an asset which can be side-loaded onto your own device.
+
+```bash
+# To build for iOS with demo credentials
+shipthis game ship --platform ios --follow --useDemoCredentials --download game.ipa
+
+# To build for Android with demo credentials
+shipthis game ship --platform android --follow --useDemoCredentials --downloadAPK game.apk
+```
+
+### Overriding the Godot version
+
+You can specify a different Godot version to use only for the current job. This can be helpful if you are upgrading your game to use a newer version of Godot.
```

Comment:
As the `shipthis.json` will still have a version we should probably make it clear here where the version actually comes from so that people can reason about whether this is needed or not.

Reply:
Good shout - those values in the file are still confusing

I have gone ahead converted the project in the config file to just an id - I need to do some more testing of the other commands that this affects.